.github/dependabot.yml

```diff
@@ -10,6 +10,53 @@ updates:
     directory: "/"
     # The maximum number of pull requests for version updates
     open-pull-requests-limit: 10
+    groups:
+      radix-ui:
+        patterns:
+          - "@radix-ui/*"
+      graphql:
+        patterns:
+          - "graphql"
+          - "@graphql-codegen/*"
+          - "@0no-co/graphqlsp"
+      eslint:
+        patterns:
+          - "eslint"
+          - "eslint-*"
+          - "@eslint/*"
+      typescript:
+        patterns:
+          - "typescript"
+          - "tsx"
+          - "@types/*"
```

Comment:
Again I do not see a reason to group these due to missing relationsships.

In general I would expect a `@types/*` packages to be placed in a group with the library it provide types for e.g. `@types/jwt-simple` and `jwt-simple`, `@types/react` and `@types/react-dom` with `react` and `react-dom` etc.